src/Sulu/Bundle/ContentBundle/Resources/public/js/main.js

```diff
@@ -37,6 +39,8 @@ define([
 
             'use strict';
 
+            SeoTab.initialize(app);
```

Comment:
no this function will be called within the `start` function. this is the way we do it always (see https://github.com/sulu/sulu/blob/develop/src/Sulu/Bundle/ContactBundle/Resources/public/js/main.js#L61)